src/core/Akka.Cluster.Tests/GossipSpec.cs

```diff
@@ -257,11 +257,11 @@ public void A_gossip_must_find_two_oldest_per_role_as_targets_for_Exiting_change
             Member a8 = TestMember.Create(new Address("akka.tcp", "sys", "a8", 2552), MemberStatus.Exiting, ImmutableHashSet<string>.Empty.Add("role1"), upNumber: 8);
             Member a9 = TestMember.Create(new Address("akka.tcp", "sys", "a9", 2552), MemberStatus.Exiting, ImmutableHashSet<string>.Empty.Add("role2"), upNumber: 9);
 
-            IEnumerable<Member> theExiting = new Member[] { a5, a6 };
+            var theExiting = new Member[] { a5, a6 };
```

Comment:
Cleaned this up to just use an `IReadOnlyCollection<Member>` - to avoid some of the multiple enumeration problems `GossipTargetsForExitingMembers` was susceptible to.